builtin/files/userdata/cloud-config-controller

```diff
@@ -3408,6 +3408,9 @@ write_files:
           {{- if .ControllerFeatureGates.Enabled }}
           - --feature-gates={{.ControllerFeatureGates.String}}
           {{- end }}
+          {{- if .Experimental.Admission.PersistentVolumeClaimResize.Enabled }}
+          - --feature-gates=ExpandPersistentVolumes=true
```

Comment:
This will clash if .ControllerFeatureGates has been set above.  It looks like the expand persistent volume claim has been on by default since 1.11 https://v1-13.docs.kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/